CoherentTasks/CoherentTasks/MaxMinArraySum.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CoherentTasks
+{
+    class MaxMinArraySum
+    {
+        public static void Execute()
+        {
+            Console.Write("Please enter your array with space between numbers: ");
+            string inputArray = Console.ReadLine();
+
+            int answer = ArrayMaxMinSum(inputArray);
+
+            Console.WriteLine($"This is your array: {inputArray}");
+            Console.WriteLine($"The sum of this array range starting from first highest number until lowest number after is {answer}");
+        }
+
+        private static int ArrayMaxMinSum(string array)
+        {
+            int[] userArray = ConvertStringArrayToIntArray(array);
+
+            int maxNumberIndex = Array.FindIndex(userArray, n => n.Equals(userArray.Max()));
+            int minNumberIndex = Array.FindLastIndex(userArray, n => n.Equals(userArray.Min()));
+
+            int arrayRangeSum = 0;
+
```

Comment:
What about if maxNumberIndex > minNumberIndex ? "outputs the sum of the array
elements located between the smallest element in the array (the leftmost element if there
are several) and the largest element (the rightmost element if there are several)."

Reply:
I see the flaw... Didn't test code in more complicated circumstances. Made all the changes required.